exercises/practice/saddle-points/runtests.jl

```diff
@@ -0,0 +1,48 @@
+using Test
+
+include("saddle-points.jl")
+
+@testset "Can identify single saddle point" begin
+    matrix = [9 8 7; 5 3 2; 6 6 7]
+    @test saddlepoints(matrix) == [(2, 1)]
```

Comment:
I opted for the return value to be a vector of tuples (and not `CartesianIndex` values) to give the student some freedom in how to approach this. Let me know if that is okay.